rust/worker/src/segment/record_segment.rs

```diff
@@ -578,4 +578,11 @@ impl RecordSegmentReader<'_> {
     ) -> Result<&str, Box<dyn ChromaError>> {
         self.id_to_user_id.get("", offset_id).await
     }
+
+    pub(crate) async fn get_offset_id_for_user_id(
```

Comment:
i think you need to merge main - this looks like a strange merge - this function already exists + more?

https://github.com/chroma-core/chroma/blob/04ad4d8058adc508abba43217569885a2288e939/rust/worker/src/segment/record_segment.rs#L588

Reply:
There was some wired thing with graphite. The merge conflict is not resolved.